src/frontends/paddle/tests/CMakeLists.txt

```diff
@@ -53,6 +53,20 @@ set(PADDLEDET_OPS_SHA256 "5cc079eda295ed78b58fba8223c51d85a931a7069ecad51c6af5c2
 
 DownloadAndCheck(${PADDLEDET_OPS_URL} ${PADDLEDET_DIRNAME}/ops.py PADDLEDET_FATAL PADDLEDET_RESULT ${PADDLEDET_OPS_SHA256})
 
+function(GET_PADDLE_VERSION output_variable)
+    execute_process(
+        COMMAND ${PYTHON_EXECUTABLE} -c "import paddle; print(paddle.__version__)"
+        RESULT_VARIABLE result
+        OUTPUT_VARIABLE ${output_variable}
+        ERROR_QUIET
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+
+    if(NOT result EQUAL "0")
+        message(FATAL_ERROR "Failed to get version of PaddlePaddle")
+    endif()
+endfunction()
```

Comment:
looks like this function is not used

Reply:
> looks like this function is not used

Yes, I will remove it in the next commit.